### PR TITLE
Adds Cloud Servers examples

### PR DESCRIPTION
**Pull request by annegentle**
I also want to merge the new index.rst from the other PR. Also will add an example of creating an instance with cloud-init.

### CODE REVIEW

File: docs/services/cloudserversexamples.rst [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+Now you can view the server to make sure the Status is ACTIVE::
+
+   $ rack servers instance list
+
+    ID                    Name        Status    PublicIPv4    PrivateIPv4    Image            Flavor
+ab95d1d6-27d1-42bb-8cdc-800efcb5fc1f    devserver    ACTIVE    23.253.50.105    10.209.137.64    09de0a66-3156-48b4-90a5-1cf25a905207    general1-4
+
+To connect to the server with SSH using your public key, use this command::
+
+    $ ssh -i ~/.ssh/id_rsa.pub root@23.253.50.104
```
Comment (annegentle) [inline docs/services/cloudserversexamples.rst]: Is this generally "how it's done?"

Comment (jrperritt) [inline docs/services/cloudserversexamples.rst]: I usually just `ssh root@23.253.50.104`

**Comment by annegentle** [comment]
Ok, this has been rebased and should be ready to go.

**Comment by jrperritt** [comment]
:+1:

**Pull request merged**